.gitignore

```diff
@@ -1,8 +1,9 @@
+/.vscode/
+/geotrans3.7/
```

Comment:
I'm not super familiar with .gitignore syntax.  What does adding the `/` in front of `geotrans3.7/` do?

Reply:
The leading slash means it only ignores the entry if it's in the root of the project.  It's a common mistake people make in my team to ignore *all* "foo/" when they meant "/foo/" so I changed it from force of habit.  If that's not your intent, sorry.

Reply:
good either way :-)